FontAwesome5.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * FontAwesome5 icon set component.
+ * Usage: <FontAwesome5 name="icon-name" size={20} color="#4F8EF7" />
+ */
+
+import React, {PureComponent} from 'react';
+
+import {Platform} from 'react-native';
+
+import PropTypes from 'prop-types';
+
+import createIconSet, {
+  DEFAULT_ICON_COLOR,
+  DEFAULT_ICON_SIZE
+} from './lib/create-icon-set';
+
+import glyphMap from './glyphmaps/FontAwesome5.json';
```

Comment:
Would prefer if the glyph maps were separated into two files and passed to the factory function.